scrnaseq.Rmd

```diff
@@ -37,26 +38,25 @@ plan("multiprocess") # by default, this uses all available cores; use "workers=4
 # Functional enrichment: enrichR
 
 # Source plotting functions
-path_to_git = "~/scrnaseq"
+path_to_git = "."
 source(paste0(path_to_git, "/R/functions_io.R"))
 source(paste0(path_to_git, "/R/functions_plotting.R"))
 source(paste0(path_to_git, "/R/functions_analysis.R"))
 source(paste0(path_to_git, "/R/functions_util.R"))
 
 # Knitr default options
-knitr::opts_chunk$set(echo=TRUE, cache=FALSE, message=FALSE, warning=FALSE, fig.width=10)
+knitr::opts_chunk$set(echo=TRUE, cache=FALSE, message=FALSE, warning=FALSE, tidy=FALSE, fig.width=10)
 
 # Potentially needed for clustering, umap, other python packages
 use_python('/usr/bin/python')
 
-# biomaRt mirror
-mirror=NULL
+# biomaRt mirror: NULL means the first working mirror is used
+mirror = NULL
 ```
 
 # Dataset description
-* 10X published dataset, downloaded from [here](https://support.10xgenomics.com/single-cell-gene-expression/datasets/3.0.0/pbmc_1k_v3) 
-* Single Cell Gene Expression Dataset by Cell Ranger 3.0.0  
-* 1k Peripheral blood mononuclear cells (PBMCs) from a healthy donor (v3 chemistry)  
+* 10x and Smartseq2 datasets of PBMC cells
+* from "Systematic comparative analysis of single cell RNA-sequencing methods" (Ding et al 2018)
```

Comment:
Can you add the reference with knit citations?

Reply:
yes